README.md

```diff
@@ -67,13 +67,9 @@ fn main() -> Result<(), String> {
                         Edn::Map(Map::new(map!{":a".to_string() => Edn::Key(":b".to_string())})),
                         Edn::Key(":A".to_string()),
                         Edn::Nil}))}
-        )))
+        ))
     );
 
-    // OR 
-
-    let edn_resp = edn_rs::from_str(edn_str)?;
```

Comment:
the ideia of this test was to show navigation on the Edn response, I think it is important to have it in the readme

Reply:
Okay, I will do the same with the above structure.

Reply:
Solved!